pkg/cmd/adm/register_member.go

```diff
@@ -509,6 +517,20 @@ until the SpaceProvisionerConfig.spec.enabled is set to true.
 `, v.hostClusterData.apiEndpoint))
 }
 
+func (v *registerMemberValidated) getRegMemConfigFlagsAndClient(ctx *clicontext.CommandContext, clusterName string) (kubeConfigFlag *genericclioptions.ConfigFlags, rccl runtimeclient.Client, err error) {
```

Comment:
the params are not used: 
```suggestion
func (v *registerMemberValidated) getRegMemConfigFlagsAndClient(_ *clicontext.CommandContext, _ string) (kubeConfigFlag *genericclioptions.ConfigFlags, rccl runtimeclient.Client, err error) {
```